My-Spec-Tutorial.package/RadioButtonGroup.extension/instance/selectedButton.st

```diff
@@ -0,0 +1,4 @@
+*My-Spec-Tutorial
+selectedButton
+
+	^ buttons detect: [ :b | b state ] ifNone: [ nil ]
```

Comment:
See how much simpler this is than that switch statement! One could easily argue that this should have been provided in the kernel, and if you open an issue, it is a good candidate for integration.